core/arch/arm/include/mm/core_mmu.h

```diff
@@ -115,6 +116,12 @@ struct core_mmu_phys_mem {
 		__used __section("phys_mem_map_section") = \
 		{ #addr, (type), (addr), (size) }
 
+#define register_sdp_mem(pa, sz) \
+	static const struct core_mmu_phys_mem __phys_sdp_mem_ ## pa\
```

Comment:
Do we need the `## pa` here? Or you think it's necessary to support more than one area? In that case, I guess the `__section("phys_sdp_mem_section")` also would need to have an area unique name?